src/views/connected/structure/DemandeReco.js

```diff
@@ -178,7 +178,7 @@ function DemandeReconventionnement() {
             </button>
           </li>
           <li>
-            <button className="fr-btn" onClick={() => setOpenModal(true)}>
+            <button disabled={!structure?.conventionnement?.dossierReconventionnement?.numero} className="fr-btn" onClick={() => setOpenModal(true)}>
```

Comment:
disabled par condition ou par code ? (pour éviter qu'un petit malin enleve le disabled dans le html)
```suggestion
structure?.conventionnement?.dossierReconventionnement?.numero && 
            <button className="fr-btn" onClick={() => setOpenModal(true)}>
```